SimPEG/Props.py

```diff
@@ -157,11 +156,16 @@ def fget(self):
             return mapping * self.model
 
         def fset(self, value):
-            value = scope.validate(self, value)
+            if value is not properties.utils.undefined:
+                value = scope.validate(self, value)
+                if scope.reciprocal:
```

Comment:
the reciprocal should only be cleared if we are not setting it to an undefined (otherwise we run into issues with defaults... by setting a reciprocal with a default, we previously cleared the default. There is now a test to check this )